docker/Dockerfile.gpu

```diff
@@ -21,7 +21,8 @@ RUN apt-get update && \
     && cd /tmp \
     && curl -O https://bootstrap.pypa.io/get-pip.py \
     && python get-pip.py \
-    && pip install gevent gunicorn mxnet-model-server \
+    && pip install gevent gunicorn \
+    && pip install --pre mxnet-model-server \
     && pip install --no-cache-dir mxnet-cu90 \
```

Comment:
I added this --no-cache-dir to avoid the build failures on docker hub as the machines ran out of resources. I don't think its required for other pip installs. Do you see any reason why it's required?

Reply:
it's a good idea... the docker images will be smaller b/c they won't contain the cached packages.